projects/media-viewer/src/lib/media-viewer.component.ts

```diff
@@ -16,9 +17,12 @@ export class MediaViewerComponent implements OnInit {
   @Input() contentType: string;
   @Input() actionEvents = new ActionEvents();
   @Input() showToolbar = true;
+  @Input() showAnnotations = false;
   @Input() showCommentSummary: Subject<boolean>;
   @Input() toolbarButtonToggles: ToolbarButtonToggles;
   @Input() commentSummaryToggle: Subject<SetCurrentPageOperation>;
+  @Input() annotations: Annotation[] = [];
```

Comment:
Not sure if you're up to date with the latest master but I think this would be better to take the AnnotationSet rather than Annotation[]